lib/spack/spack/patch.py

```diff
@@ -350,13 +371,20 @@ def from_dict(
             dictionary["url"],
             dictionary["level"],
             working_dir=dictionary["working_dir"],
+            # Added recently, fallback required for backwards compatibility
```

Comment:
```suggestion
            # Added in v0.22, fallback required for backwards compatibility
```

Reply:
Just spotted these in email, I think it would be better to give a version reference for future readers